compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -1223,4 +1223,22 @@ object messages {
            |Attempting to define a field in a method signature after a varargs field is an error.
            |""".stripMargin
   }
+
+  case class MethodDoesNotTakeParameters(tree: tpd.Tree, methPartType: Types.Type)(err: typer.ErrorReporting.Errors)(implicit ctx: Context)
+  extends Message(MethodDoesNotTakeParametersId) {
+    private val more = tree match {
+      case Apply(_, _) => " more"
+      case _ => ""
+    }
+
+    val msg = hl"${err.refStr(methPartType)} does not take$more parameters"
+
+    val kind = "Reference"
+
+    val explanation =
+      hl"""|You have specified more parameter lists as defined in the method definition(s).
+           |In case ${err.refStr(methPartType)} is defined without parenthesis, you may
```

Comment:
@smarter I find a `TermRefWithSignature` in my samples, do I need to test for just that?